linode/manager
# Added the webdriverio spec reporter, for clearer failed test reporting

Added wdio-spec-reporter for clearer reporting in CI run.

**To test**
`yarn && yarn start` new shell `yarn selenium` new shell `yarn e2e --log --spec=e2e/specs/account/view-update-permissions.spec.js` ensure reporting output should contain test step detail.
